src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -128,6 +128,12 @@
 
     public VehicleValidator vehicleValidator = new VehicleValidator();
 
+    /**
+     * Threshold for deciding when vehicle is assumed to be missing when using vehiclePresencePredictor functionality
+     * Default 0 means it is always present.
+     */
+    public float vehiclePredictionThreshold = 0;
```

Comment:
`private` with getter and setter or `public` without them

Reply:
let it be public and wait for the day of refactor